world-traditional-food.md

```diff
@@ -47,4 +48,20 @@ Rendang is traditionally served at ceremonial occasions to honour guests during
 
 <img src="https://indonesiakaya.com/wp-content/uploads/2023/04/ren_Artboard_4.jpg" width="400" alt="rendang indonesian food">
 
+### Philippines
+
+#### Adobong Manok
+
+Adobong manok, or "chicken adobo," is widely regarded to be the national dish of the Philippines. The beauty of chicken adobo is not just the savory flavors of the dish but also how simple and easy it is to prepare. 
+
+Every cook might have their own methods for cooking but the recipe typically require only a few basic ingredients that are commonly available in most kitchens: vinegar, soy sauce, black peppercorns, and bay leaves. 
+
+The cooking process usually involves marinating the chicken in a mixture of these ingredients in a pot for a short period. After marinating, the chicken is simmered in the marinade until tender. This dish is traditionally served with steamed rice or garlic fried rice.
+
+<img src="images/chicken-adobo.jpeg" width="400" alt="rendang indonesian food">
```

Comment:
Change the alt text to your tradional food.

Reply:
Changed alt text from "rendang indonesian food" to "chicken adobo filipino food"